packages/federation-sdk/src/services/staging-area.service.ts

```diff
@@ -365,6 +367,35 @@ export class StagingAreaService {
 						},
 					});
 					break;
+				case EventType.REACTION: {
+					this.eventEmitterService.emit('homeserver.matrix.reaction', {
+						event_id: event.eventId,
+						room_id: event.roomId,
+						sender: event.event.sender,
+						origin_server_ts: event.event.origin_server_ts,
+						content: event.event.content as {
+							'm.relates_to': {
+								rel_type: 'm.annotation';
+								event_id: string;
+								key: string;
+							};
+						},
+					});
+					break;
+				}
+				case EventType.REDACTION: {
```

Comment:
Don't we need to find a way to know if the event is an unreaction OR message deletion?

Reply:
agreed